charts/shoot-core/components/charts/monitoring/charts/node-exporter/templates/node-exporter.yaml

```diff
@@ -29,6 +29,8 @@ metadata:
   name: node-exporter
   namespace: kube-system
   labels:
+    networking.gardener.cloud/to-public-networks: allowed
```

Comment:
Node exporter doesn't need to talk to any component? I don't think this is needed

Reply:
this line was added per your previous comments https://github.com/gardener/gardener/pull/2044#discussion_r392054792
Thanks!

Reply:
I was mistaken I meant `metric server`... Sorry for the confusion.

Reply:
Hi @mvladev , currently in `metrics-server\deployment.yaml` there are two network policies
```
    networking.gardener.cloud/from-seed: allowed
    networking.gardener.cloud/to-dns: allowed
```
Sorry i'm confused do i still need to modify this file?
Thanks.

Reply:
` networking.gardener.cloud/to-dns: allowed` is not needed.

To summarize: 

1. node-exporter needs to receive traffic only from the VPN
1. metric server needs to receive traffic only from the VPN
1. metric server needs to send traffic to the DNS,  Nodes (their CIDR) and the API server.

Reply:
thanks - i've removed this line